Add configuration for new notifications check (#13049)
pimcore/pimcore: doc/Development_Documentation/18_Tools_and_Features/41_Notifications.md

}
```

### Configuration

> The configuration allow to enable/disable the *Notifications menu* in the toolbar
> and the *Check of new notifications* (url: /admin/notification/find-last-unread).
> It takes place in the `pimcore_admin` configuration within the section `notifications`.

Reviewer: ```suggestion
The configuration allows enabling/disabling the *Notifications menu* in the toolbar
and the *check of new notifications* (url: `/admin/notification/find-last-unread`).
It is done in the `pimcore_admin` configuration in the `notifications` section.
```

default:
```yaml
pimcore_admin:
    notifications:
        enabled: true
        check_new_notification:
            enabled: true
            interval: 30
```

#### Example configurations:

* Disable notifications menu and check of new unread notifications:
```yaml
pimcore_admin:
    notifications:
        enabled: false
```

* Disable only the check of new unread notifications:
```yaml
pimcore_admin:
    notifications:
        check_new_notification:
            enabled: false
```

> * Change interval to get new unread notifications:

Reviewer: ```suggestion
* Change interval to fetch new unread notifications (in seconds):
```
